plugins/modules/ec2_vpc_peering_info.py

```diff
@@ -381,16 +244,7 @@ def main():
     for peer in results:
         peer["tags"] = boto3_tag_list_to_ansible_dict(peer.get("tags", []))
 
-    module.deprecate(
-        (
-            "The 'result' return key is deprecated and will be replaced by 'vpc_peering_connections'. Both values are"
-            " returned for now."
-        ),
-        version="11.0.0",
-        collection_name="amazon.aws",
-    )
-
-    module.exit_json(result=results, vpc_peering_connections=results)
```

Comment:
Can this be deprecated during 11.0.0 release? The tests will fail with this change

Reply:
@gravesm I have added a fix to the test. Can you please take a look?